app/services/feeds/import.rb

```diff
@@ -1,13 +1,24 @@
 module Feeds
+  # Responsible for fetching RSS feeds for multiple users.
+  #
+  # @see {Feeds::Import.call}
   class Import
-    def self.call(users: nil, earlier_than: nil)
-      new(users: users, earlier_than: earlier_than).call
+    # Fetch the feeds for the given users (with some filtering based on internal business logic).
+    #
+    # @param users_scope [ActiveRecord::Relation<User>] the initial scope for determining the users
+    #        whose feeds we'll be fetching.
+    #
+    # @param earlier_than [NilClass, ActiveSupport::TimeWithZone] when given, use this to further
+    #        filter the user's who's articles we'll fetch.  That is to say, we won't fetch anyone's
+    #        feeds who's last fetch time was after our earlier_than parameter.
+    #
+    # @return [Integer] count of total articles fetched.
+    def self.call(users_scope: User, earlier_than: nil)
+      new(users_scope: users_scope, earlier_than: earlier_than).call
     end
 
-    def initialize(users: nil, earlier_than: nil)
-      # using nil here to avoid an unnecessary table count to check presence
-      @users = users || User.where(id: Users::Setting.with_feed.select(:user_id))
-      @earlier_than = earlier_than
+    def initialize(users_scope: User, earlier_than: nil)
```

Comment:
Normally I like the conciseness of just calling it `users`, because it _is_ an `Enumerable` that yields `User` instances, but I think this change clarifies that that's not actually what it wants and instead that it wants takes in the relation because it needs to scope it down more. :100: